examples/chatgpt/rag-quickstart/azure/Azure_AI_Search_with_Azure_Functions_and_GPT_Actions_in_ChatGPT.ipynb

```diff
@@ -0,0 +1,1330 @@
+                "vector_query = VectorizedQuery(vector=generate_embeddings(query, embeddings_model), k_nearest_neighbors=3, fields=\"content_vector\")\n",
+                "  \n",
+                "results = search_client.search(  \n",
+                "    ## Notice search_text is query below, as opposed to None in the prior cell.\n",
+                "    search_text=query,  \n",
+                "    vector_queries= [vector_query], \n",
+                "    select=[\"title\", \"text\"],\n",
+                "    filter=f\"category eq '{category}'\",\n",
+                "    top=3\n",
+                ")\n",
+                "\n",
+                "for result in results:  \n",
+                "    print(result)"
+            ]
+        },
+        {
+            "cell_type": "markdown",
+            "metadata": {},
+            "source": [
+                "## Creating an Azure Function to Search\n",
+                "\n",
+                "Azure Functions are an easy way to build an API on top of our new AI search. In our code (see the `function_app.py` file in this folder, or linked [here](https://github.com/openai/openai-cookbook/blob/main/examples/chatgpt/rag-quickstart/azure/function_app.py)) it does the following:\n",
+                "\n",
+                "1. Takes in an input of the user's query, search index endpoint, the index name, the k_nearest_neighbors, the search column to use (either content_vector or title_vector), and whether it should use a hybrid query\n",
+                "\n",
+                "2. Takes the user's query and embeds it.\n",
+                "\n",
+                "3. Conducts a vector search and retrieves relevant text chunks.\n",
+                "\n",
+                "4. Returns those relevant text chunks as the response body. \n",
+                "\n",
+                "**  In the context of vector search, k_nearest_neighbors specifies the number of \"closest\" vectors (in terms of cosine similarity) that the search should return. For example, if k_nearest_neighbors is set to 3, the search will return the 3 vectors in the index that are most similar to the query vector.\n",
+                "\n",
+                "> Note that this Azure Function _does not have any authentication_. However, you can set authentication on it following docs [here](https://learn.microsoft.com/en-us/azure/azure-functions/security-concepts?tabs=v4)"
+            ]
+        },
+        {
+            "cell_type": "markdown",
+            "metadata": {},
+            "source": [
+                "### Create a storage account"
+            ]
+        },
+        {
+            "cell_type": "markdown",
+            "metadata": {},
+            "source": [
+                "We'll create a storage account using the code below. This walkthrough creates a new storage account, but feel free to modify to use existing. This may take up to 30 seconds"
```

Comment:
```suggestion
                "We'll create a storage account using the code below. This walkthrough creates a new storage account, but feel free to modify to use existing. This may take up to 30 seconds."
```